lib/rpmsg/rpmsg_virtio.c

```diff
@@ -636,11 +639,20 @@ static int rpmsg_virtio_ns_callback(struct rpmsg_endpoint *ept, void *data,
 	if (ns_msg->flags & RPMSG_NS_DESTROY) {
 		if (_ept)
 			_ept->dest_addr = RPMSG_ADDR_ANY;
+		if (_ept && _ept->release_cb) {
+			rpmsg_ept_incref(_ept);
+			ept_decref = true;
```

Comment:
useless local variable

Reply:
because some serives will free ept in _ept->ns_unbind_cb， if not added ept_decref 
```
if (_ept && _ept->ns_unbind_cb)
   _ept->ns_unbind_cb(_ept);
metal_mutex_acquire(&rdev->lock);
rpmsg_ept_decref(_ept);
metal_mutex_release(&rdev->lock);
```
rpmsg_ept_decref will use-afrer-free